samples/unit-tests/3d/column-stacked-labels/demo.js

```diff
@@ -1,3 +1,33 @@
+QUnit.test("JS error when stack ID's are strings in Highcharts 3D.(#4532)", function (assert) {
```

Comment:
This test doesn't concern stack _labels_, so it should be moved to a new file `unit-tests/3d/series`.